metadata-ingestion/src/datahub/ingestion/source/unity/config.py

```diff
@@ -12,9 +13,12 @@
 from datahub.ingestion.source.state.stateful_ingestion_base import (
     StatefulIngestionConfigBase,
 )
+from datahub.ingestion.source.usage.usage_common import BaseUsageConfig
 
 
-class UnityCatalogSourceConfig(StatefulIngestionConfigBase, DatasetSourceConfigMixin):
+class UnityCatalogSourceConfig(
+    StatefulIngestionConfigBase, BaseUsageConfig, DatasetSourceConfigMixin
```

Comment:
BaseUsageConfig has a lot of fields - if not all of them are supported, then we should change the base class that we're using

Reply:
We use most of them, as they get used by `usage_common`. The only one that doesn't get used is `include_read_operational_stats` which seems to only be used by bigquery... so if anything I say we remove it from this common config and put it only in the bigquery one.